app/controllers/concerns/page_tool_tip.rb

```diff
@@ -0,0 +1,22 @@
+module PageToolTip
+  def set(collection, model_name)
```

Comment:
Love this refactor 💯 
Two things:
- Should we rename PageToolTip to CollectionPageSetup, since it's setting up more than just the tool tip?
- Instead of passing in model_name should we pass the model itself?